pkg/server/hook_handler.go

```diff
@@ -3,14 +3,14 @@ package server
 import (
 	"context"
 	"fmt"
+	"github.com/zapier/kubechecks/pkg/config"
```

Comment:
`"github.com/zapier/kubechecks/pkg/config"` should be on the same group as the rest